packages/button/src/mwc-button-base.ts

```diff
@@ -32,6 +32,8 @@ export class ButtonBase extends LitElement {
 
   @property({type: Boolean}) trailingIcon = false;
 
+  @property({type: String}) type = 'submit';
```

Comment:
I think the in the html spec the default type is `button`

Also might be good to add a typescript string literal type for this
`type ButtonType = 'button' | 'submit' | 'reset'`

resulting in 
`@property({type: String}) type: ButtonType = 'button';`

Reply:
FYI the default type in HTML is `submit`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Reply:
sorry about that, yes `submit` is the default